addons/centered/src/react.tsx

```diff
@@ -5,6 +5,17 @@ import parameters from './parameters';
 import styles from './styles';
 
 function centered(storyFn: () => ReactNode) {
+  /* eslint-disable no-undef */
```

Comment:
Perhaps a `useViewmode` hook would be a good idea?

Reply:
@ndelangen I think that would be very useful, independent of this PR